website/docs/helpers.mdx

```diff
@@ -4,55 +4,67 @@ title: Helpers
 sidebar_label: Helpers
 ---
 
-### `createNavigationCommands`
+### `setRoot`
 
-A set of overloaded methods which reduce the boilerplate required to execute common navigation commands.
+Overloaded method which reduce the boilerplate required for the `setRoot` command.
 
 ```ts
 interface SetRootCommand {
   (layoutRoot: LayoutRoot): Promise<any>
   (layout: Layout): Promise<any>
   <P = {}>(name: string, passProps?: P, options?: Options): Promise<any>
 }
+```
 
-interface SetStackRootCommand {
+### `showModal`
+
+Overloaded method which reduce the boilerplate required for the `showModal` command.
+
+```ts
+interface ShowModalCommand {
   (layout: Layout): Promise<any>
   <P = {}>(name: string, passProps?: P, options?: Options): Promise<any>
 }
+```
 
-interface PushCommand {
-  <P = {}>(layout: Layout<P>): Promise<any>
-  (name: string): Promise<any>
-  <P = {}>(name: string, passProps?: P): Promise<any>
+### `showOverlay`
+
+Overloaded method which reduce the boilerplate required for the `showOverlay` command.
```

Comment:
And here

Reply:
thanks! fixed in 93c6fc57642804cf6f30db5841a139349273b188